Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml

```diff
@@ -0,0 +1,392 @@
+      - "2"
+    scriptarguments:
+      details: {}
+      fromclosedate: {}
+      fromdate: {}
+      fromduedate: {}
+      id: {}
+      level: {}
+      name: {}
+      notstatus: {}
+      owner: {}
+      page: {}
+      query:
+        simple: ${inputs.IncidentSearchQuery}
+      reason: {}
+      size: {}
+      sort: {}
+      status: {}
+      toclosedate: {}
+      todate: {}
+      toduedate: {}
+      type: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "2":
+    id: "2"
+    taskid: 6aadb57e-7e9f-4f73-88ea-5076e7918d07
+    type: condition
+    task:
+      id: 6aadb57e-7e9f-4f73-88ea-5076e7918d07
+      version: -1
+      name: Are there open incidents?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: foundIncidents
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "3":
+    id: "3"
+    taskid: def4dada-8825-435d-8fdf-9e922185a27a
+    type: regular
+    task:
+      id: def4dada-8825-435d-8fdf-9e922185a27a
+      version: -1
+      name: Assign Incidents to next shift
+      description: |-
+        Randomly assigns the incidents to users on call (requires shift management). This is an example automation for reassigning a large number of incidents to the next shift. This automation works with the  Out of Office automation to ensure only available users get assigned.
+
+        https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/users-and-roles/shift-management.html#idf554fd0f-f93b-40cd-9111-1393bf25ac6e
+
+        Incident IDs should be passed in a comma-separated list.
+      scriptName: AssignToNextShiftOOO
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      incident_id:
+        simple: ${foundIncidents.id}
+      listname: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 520,
+          "y": 945
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: c9aebe9c-0692-4f6c-8454-157709aa68b8
+    type: regular
+    task:
+      id: c9aebe9c-0692-4f6c-8454-157709aa68b8
+      version: -1
+      name: Get Users on Call
+      description: commands.local.cmd.get.users
```

Comment:
@ShirleyDenkberg this is a server default description and not the correct description.